js/private/js_binary.bzl

```diff
@@ -247,6 +247,12 @@ _ATTRS = {
         # TODO(2.0): make this mandatory so that downstream binary rules that inherit these attributes are required to set it
         mandatory = False,
     ),
+    "node_toolchain": attr.label(
+        doc = """The nodejs toolchain to use for this target.
```

Comment:
nit: Add that if none is set then Bazel will select the toolchain based on the platform settings.

Reply:
Added that in since you probably haven't had your first cup of coffee yet :)

Reply:
Oh nevermind. I can't push to your fork. Here is my suggestion,
```
    "node_toolchain": attr.label(
        doc = """The Node.js toolchain to use for this target.

        See https://bazelbuild.github.io/rules_nodejs/Toolchains.html

        Typically this is left unset so that Bazel automatically selects the right Node.js toolchain
        for that target platform. See https://bazel.build/extending/toolchains#toolchain-resolution
        for more information.
        """,
    ),
```